packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -7,13 +7,14 @@ import { EventEmitter } from "events";
 import { stringToBuffer } from "@fluid-internal/client-utils";
 import { IIdCompressor, IIdCompressorCore, IdCreationRange } from "@fluidframework/id-compressor";
 import { assert } from "@fluidframework/core-utils";
-import { ITelemetryLoggerExt, createChildLogger } from "@fluidframework/telemetry-utils";
+import { createChildLogger } from "@fluidframework/telemetry-utils";
```

Comment:
Looks like this import isn't necessary. The build is probably complaining about it?